### PR TITLE
Added proper route metrics.

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

File: app/com/arpnetworking/play/metrics/MetricsActionWrapper.java [hunk 1]
```diff
@@ -62,17 +81,28 @@ public MetricsActionWrapper(final MetricsFactory metricsFactory) {
...
-    protected String createTimerName(final Http.Context context) {
-        // TODO(vkoskela): Use routes information to replace variable path parts with variable names. [MAI-280]
+    protected static String createRouteMetricName(final Http.Context context) {
         final Http.Request r = context.request();
         final StringBuilder metricNameBuilder = new StringBuilder("rest_service/");
         metricNameBuilder.append(r.method());
 
-        if (!Strings.isNullOrEmpty(r.path())) {
-            if (!r.path().startsWith("/")) {
+        final String route = context._requestHeader().tags().get("ROUTE_PATTERN").getOrElse(
```
Comment (BrandonArp) [inline app/com/arpnetworking/play/metrics/MetricsActionWrapper.java]: Actually, there is a constant defined for this. Router.Tags.ROUTE_PATTERN

Comment (vjkoskela) [inline app/com/arpnetworking/play/metrics/MetricsActionWrapper.java]: Nice. Done.

Comment (vjkoskela) [inline app/com/arpnetworking/play/metrics/MetricsActionWrapper.java]: Yes! Done.

File: app/com/arpnetworking/play/metrics/MetricsActionWrapper.java [hunk 2]
```diff
@@ -62,17 +81,28 @@ public MetricsActionWrapper(final MetricsFactory metricsFactory) {
...
+                new AbstractFunction0<String>() {
+                    @Override
+                    public String apply() {
+                        return "";
+                    }
+                })
+                .replaceAll("\\$([^<]+)<([^>]+)>", ":$1");
```
Comment (vjkoskela) [inline app/com/arpnetworking/play/metrics/MetricsActionWrapper.java]: Haha, really? It's going to be called that often ;) -- But yes, you are correct. Done.

**Comment by vjkoskela** [comment]
@BrandonArp updated.

**Pull request merged**